ecl/eclagent/eclagent.cpp

```diff
@@ -2107,6 +2095,8 @@ void EclAgent::doProcess()
         }
         e->Release();
     }
+    wuRead.clear(); // have a write lock still, but don't want to leave dangling unlocked wuRead after releasing write lock
```

Comment:
@jakesmith please look again.
Do you understand this comment (which was already there)?  The comment is no longer true, so should be deleted, but is it something we should be worrying about?

Reply:
I don't understand that comment, should remove it.
I don't think this wuRead.clear() is needed either.